src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -56,7 +57,8 @@ internal class SqlServerSearchService : SearchService
         private readonly SchemaInformation _schemaInformation;
         private readonly ICompressedRawResourceConverter _compressedRawResourceConverter;
         private readonly RequestContextAccessor<IFhirRequestContext> _requestContextAccessor;
-        private const int _resourceTableColumnCount = 10;
+        private const int _defaultResourceTableFinalSelectColumnCount = 11;
```

Comment:
>Final [](http://example.com/codeflow?start=47&length=5)

What `Final` means?

Reply:
I meant to refer to the "last" SELECT we do on the Resource table before returning the results. Maybe `Final` is not the right word to use...

Reply:
Updated name to `_defaultNumberOfColumnsReadFromResult`